### PR TITLE
Document Vale

### PR DESCRIPTION
**Pull request by jdbaldry**

### CODE REVIEW

**Review by knylander-grafana — APPROVED** [review]
Thank you for adding doc. The updates look good.

File: docs/sources/writing-guide/tooling-and-workflows/lint-prose/index.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
+---
+description: How to lint prose with the Vale linter.
+menuTitle: Lint prose
+title: Lint prose with the Vale linter
+---
+
+# Lint prose with the Vale linter
+
+[Vale](https://github.com/errata-ai/vale) is syntax-aware linter for prose built with speed and extensibility in mind.
```
Comment (josmperez) [inline docs/sources/writing-guide/tooling-and-workflows/lint-prose/index.md:9]:
```suggestion
[Vale](https://github.com/errata-ai/vale) is a syntax-aware linter for prose built with speed and extensibility in mind.
```

**Review by josmperez — APPROVED** [review]
Approved with a tiny missed word on one line. My one suggestion would be to expand on the opening where you tell people what Vale is all about. As a tech writer, I get that it lints prose, but I'm not sure where it fits into my workflow, if I need to run this, or if it gets automatically run.

File: docs/sources/writing-guide/tooling-and-workflows/lint-prose/index.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+  docs             Serve documentation locally, which includes pulling the latest `DOCS_IMAGE` (default: `grafana/docs-base:latest`) container image. See also `docs-no-pull`.
+  docs-no-pull     Serve documentation locally without pulling the `DOCS_IMAGE` (default: `grafana/docs-base:latest`) container image.
+  docs-debug       Run Hugo web server with debugging enabled. TODO: support all SERVER_FLAGS defined in website Makefile.
+  doc-validator    Run docs-validator on the entire docs folder.
+  doc-validator/%  Run doc-validator on a specific path. To lint the path /docs/sources/administration, run 'make doc-validator/administration'.
+  docs.mk          Fetch the latest version of this Makefile from Writers' Toolkit.
+```
+
+To lint prose with Vale, run `make vale` from the `docs/` directory.
+Any linting errors are logged by the tool.
```
Comment (jdbaldry) [inline docs/sources/writing-guide/tooling-and-workflows/lint-prose/index.md:36]:
```suggestion
Any linting errors are logged by the tool.

Additionally, some repositories run Vale as part of Continuous Integration (CI).
Repositories that run Vale in CI include:
- [Writers' Toolkit](https://github.com/grafana/writers-toolkit)
```

**Comment by jdbaldry** [comment]
I've added some information about the use of Vale in CI in https://github.com/grafana/writers-toolkit/pull/259/commits/137b66fc54fb0782a33c74bea26aa9da454feed0

**Pull request merged**